services/src/server.rs

```diff
@@ -112,7 +114,11 @@ where
             .configure(handlers::wcs::init_wcs_routes::<C>)
             .configure(handlers::wfs::init_wfs_routes::<C>)
             .configure(handlers::wms::init_wms_routes::<C>)
-            .configure(handlers::workflows::init_workflow_routes::<C>);
+            .configure(handlers::workflows::init_workflow_routes::<C>)
+            .service(
+                SwaggerUi::new("/swagger-ui/{_:.*}")
+                    .url("/api-doc/openapi.json", ApiDoc::openapi()),
```

Comment:
Is this name part of a standard? If not, we should camelCase it.

Reply:
> The OpenAPI Wiki [recommends](https://github.com/OAI/OpenAPI-Specification/wiki#server) using an /api-docs endpoint, at least for server APIs. I've seen several sites in the wild that use that, and it's our shop standard.

(https://stackoverflow.com/questions/41660658/openapi-or-swagger-json-auto-discovery)

it seems to be the default. We could just put it under /openapi.json directly. or /doc/openapi.json. If we do a non-standard path anyway I would go with one of the two instead of apiDocs

Reply:
Okay, but then we should do it at `/api-docs` and not `/api-doc`, right?

Reply:
2692ca2c8fa4cdc131ad683c53c7654ab5a6259d